client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,223 @@
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+];
 
 class Settings extends React.Component {
 
+	constructor( props ) {
+		super( props );
+	}
+
+	fetchSettings = ( props ) => {
+		const { siteId, fetchSettings } = props;
+		siteId && fetchSettings( siteId );
+	}
+
 	componentDidMount = () => {
-		const { siteId, fetchEmailSettings: fetch } = this.props;
-		siteId && fetch( siteId );
+		this.fetchSettings( this.props );
 	};
 
 	componentWillReceiveProps = newProps => {
-		if ( newProps.siteId === this.props.siteId ) {
-			return;
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.fetchSettings( newProps );
 		}
+	};
+
+	notificationsToggle = () => {
+		undefined;
+	}
 
-		const { siteId, fetchEmailSettings: fetch } = newProps;
-		siteId && fetch( siteId );
+	recipientsChange = () => {
+		undefined;
 	}
 
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
+		const { loading, loaded, settings } = this.props;
+		const waiting = ! loaded || loading;
+
 		return (
 			<div className="email-settings__container">
+				<div className="email-settings__origin">
+					<Card className="email-settings__origin-title">
+						Origin
+					</Card>
+					<Card className="email-settings__origin-settings">
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromName }
+								recipient={ settings.email[ fromName.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromAddress }
+								recipient={ settings.email[ fromAddress.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+					</Card>
+				</div>
+				<div className="email-settings__internal-notifications">
+					<Card className="email-settings__internal-notifications-title">
+						<div className="email-settings__section-title-text">
+							{ translate( 'Internal notifications' ) }
+						</div>
+						<div className="email-settings__section-subtitle-text">
+							{ translate( 'Email notifications sent to store staff.' ) }
+						</div>
+					</Card>
+					<Card className="email-settings__internal-notifications-legend">
```

Comment:
Instead of more cards, here we should be using `List`, `ListItem`, etc `ListHeader` components, so we can be consistent with other areas.

See https://github.com/Automattic/wp-calypso/blob/add/email-settings-ui/client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-list.js#L137-L149 for an example.